### ISSUE TITLE
Return correct mimetype for MP2 files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dhamaniasad/mutagen#163]
```
Mutagen 1.21 returns a mimetype of audio/mp3 for both MP2 and MP3. For example:

mutagen-inspect Audio_Promo_Night.mp2

identifies the file correctly but returns the wrong mimetype:

-- Audio_Promo_Night.mp2
- MPEG 1 layer 2, 128000 bps, 44100 Hz, 29.45 seconds (audio/mp3)

Two small commits in http://code.google.com/r/danieljames-mp2/source/list fix 
this for me with mutagen-inspect.
```

Original issue reported on code.google.com by `daniel.j...@sourcefabric.org` on 18 Nov 2013 at 5:03

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the late response.

I'd prefer the attached fix, since MP3 handles all of MPEG.

Any thoughts?
```

Original comment by `reiter.christoph@gmail.com` on 19 Dec 2013 at 5:33
- Changed state: **Accepted**

Attachments:
- [163-v1.patch](https://storage.googleapis.com/google-code-attachments/mutagen/issue-163/comment-1/163-v1.patch)

**Comment by GoogleCodeExporter** [comment]
```
Hi Christoph, this is a much better solution than my patch. The correct 
mimetype is returned even when the MP2 file has an incorrect or ambiguous 
extension:

('--', 'unknown-MFM0001-256kbps.mp2')
- MPEG 1 layer 2, 256000 bps, 44100 Hz, 257.16 seconds (audio/mp2)

('--', 'unknown-MFM0001-256kbps.mp3')
- MPEG 1 layer 2, 256000 bps, 44100 Hz, 257.16 seconds (audio/mp2)

('--', 'unknown-MFM0001-256kbps.mpeg')
- MPEG 1 layer 2, 256000 bps, 44100 Hz, 257.16 seconds (audio/mp2)

Thanks!
```

Original comment by `daniel.j...@sourcefabric.org` on 30 Dec 2013 at 3:22

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a51f6e192201.
```

Original comment by `reiter.christoph@gmail.com` on 9 Jan 2014 at 2:53
- Changed state: **Fixed**
